### DISCUSSION TITLE
Future Ready Talent Project

### DISCUSSION THREAD

**Discussion by jenil2401** [github/india#705] [Students]
Submitted the project and deployed it using the azure app service. Today got a mail that $20 has been used till now. And I have got only $100 for my azure account. And the results will be shown after 15 days. So what should be done in case when my project will be reviewed till that time if my credits expire it would be rejected. So requesting the mentors to guide on this or review the project as soon as if possible. @ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4

**Comment by Prachi7249** [comment]
hey, how should I complete the project ? I have completed all my self learning module. can you please please reply and help me? can you please also share your email? @jenil2401 @suniladsul26 guys please help

**Reply by jenil2401** [reply]
Yes would be helping no worries. Just share the problem you are facing on my mail id : jenilmalkan6@gmail.com
